modules/widgets/src/lib/components/grouped-icon-button.tsx

```diff
@@ -3,22 +3,26 @@
 // Copyright (c) vis.gl contributors
 
 export type GroupedIconButtonProps = {
-  className?: string;
   label: string;
+  /** Icons can be loaded from style sheet using class name */
+  className: string;
+  /** Alterhnatively an SVG icon element can be provided */
+  icon?: () => JSX.Element;
```

Comment:
I learned today that this should should just be a `JSX.Element`.

```suggestion
  icon?: JSX.Element;
```

The reason the function seemed to be needed was that defining JSX on the module level gets immediately executed by node tests as React code instead of Preact code due to the way our tests are configured. We could fix the configuration.. but in the meantime we should just type our preact code correctly and wrap all JSX in a function.

https://github.com/visgl/deck.gl/pull/9797/commits/62fe29357b3957254c1056048b80db8743d25175